data/web/inc/vars.inc.php

```diff
@@ -42,11 +42,13 @@
 $MAILCOW_APPS = array(
   array(
     'name' => 'SOGo',
-    'link' => '/SOGo/'
+    'link' => '/SOGo/',
+    'description' => 'SOGo is a collaborative software (groupware) server.'
```

Comment:
Change the descriptions to something less technical, e.g. "Web access to your email, contacts and calendar" and "Web access to your email".